test/expected/plan_ordered_append-10.out

```diff
@@ -764,10 +764,10 @@ LIMIT 2;
                ->  Index Scan using _hyper_3_7_chunk_dimension_last_time_idx on _hyper_3_7_chunk
          ->  Materialize
                ->  Append
```

Comment:
the order changed here. I guess any order is technically correct here but the new order is more efficient? Can @svenklemm confirm?

Reply:
also will this append later become a chunk append too? What's the plan

Reply:
it will be chunk append later but it wont be ordered append because of the nested loop join

Reply:
the reason why the order changes is because expand_hypertable thinks this is gonna become ordered append, but because postgres  is doing nested loop join it never creates the mergeappend plan that would be transformed into ordered append, efficiency-wise it shouldnt matter and once the append becomes a chunk append runtime exclusion should kick in and the nested loop join should only ever target 1 chunk per iteration